submission/stub.go

```diff
@@ -98,3 +98,7 @@ func buildRootedStubLC(log *loglist.Log, rCerts map[string][]rootInfo) (client.A
 func buildEmptyStubLogClient(log *loglist.Log) (client.AddLogClient, error) {
 	return buildRootedStubLC(log, map[string][]rootInfo{})
 }
+
+func buildStubLC(log *loglist.Log) (client.AddLogClient, error) {
```

Comment:
nit: `new` is a more Go conventional verb for functions creating objects. Also, `LC` is not readable (for me). Generally, the convention is `[nN]ewTypeName`. In this case I suggest `newStubLogClient`.

Reply:
Created #500